types/foundry/client/canvas/layers/base/placeables-layer.d.mts

```diff
@@ -363,27 +363,27 @@ export default class PlaceablesLayer<TPlaceable extends PlaceableObject> extends
      * @internal
      */
     _createPreview(
-        createData: DeepPartial<TPlaceable["document"]["_source"]>,
+        createData: DeepPartial<TObject["document"]["_source"]>,
         options?: { renderSheet?: boolean; top?: number; left?: number },
-    ): Promise<TPlaceable>;
+    ): Promise<TObject>;
 
     /* -------------------------------------------- */
     /*  Event Listeners and Handlers                */
     /* -------------------------------------------- */
 
-    protected override _onClickLeft(event: PIXI.FederatedEvent): void;
+    protected override _onClickLeft(event: PlaceablesLayerPointerEvent<TObject>): void;
 
-    protected override _canDragLeftStart(user: User, event: PIXI.FederatedEvent): boolean;
+    protected override _canDragLeftStart(user: User, event: PlaceablesLayerPointerEvent<TObject>): boolean;
 
-    protected override _onDragLeftStart(event: PIXI.FederatedEvent): void;
+    protected override _onDragLeftStart(event: PlaceablesLayerPointerEvent<TObject>): void;
```

Comment:
This isn't a PointerEvent

Reply:
I lifted that from `master`. I checked the actual event and `instanceof PIXI.FederatedPointerEvent` returns `true`. They also include `pointerId` and `pointerType`.